### PR TITLE
Tire tracks

### PR DESCRIPTION
**Pull request by arturkamieniecki**
Reopen #160

### CODE REVIEW

File: Project/Assets/Factory/Warehouse_300m2/Textures/WarehouseWindow/WarehouseWindow_BaseColor.png
Comment (michalpelka) [inline Project/Assets/Factory/Warehouse_300m2/Textures/WarehouseWindow/WarehouseWindow_BaseColor.png:1]: large file @pawelbudziszewski is it necessary?

Comment (pawelbudziszewski) [reply Project/Assets/Factory/Warehouse_300m2/Textures/WarehouseWindow/WarehouseWindow_BaseColor.png:1]: I guess not. It will be resized

**Comment by pawelbudziszewski** [comment]
This PR requires some updates. Converted to draft

**Comment by pawelbudziszewski** [comment]
It will be finished when  #191 is merged

**Comment by pawelbudziszewski** [comment]
replaced by #209

**Pull request closed without merging**